packages/express-openapi/index.ts

```diff
@@ -255,3 +275,25 @@ function toPromiseCompatibleMiddleware(fn) {
   }
   return fn;
 }
+
+function getBasePathsFromServers(servers: OpenAPIV3.ServerObject[]): string[] {
```

Comment:
i kind of feel like this _should_ be in `openapi-framework`.  this logic doesn't seem specific to express, and `koa-openapi` would benefit from being passed the basePaths.  Perhaps `openapi-framework` should pass `basePaths` as an array instead of `basePath` as a string?

Reply:
That is where I started with it.  I was going to expose an `baseUrls` attribute or something from the `ctx` object.  I thought to move it into express specifically because I was starting to convert the OpenApi param syntax `/{var}` to express specific syntax `/:var`.  

I guess `koa-router` also uses express routing syntax, but then `openapi-framework` would be kind of glued to that style of routing.  For example I'm not sure what if there are any plans to support hapi.js at all but they use path params like: `'/hello/{user}'`

Although I guess that being said OpenApi picked one way, we converted it to express style, if `openapi-framework` exposes express style no reason someone couldn't just convert them to whatever they needed.

Reply:
yea, i think it's find if openapi-framework exposes the urls in one format or another, but i do feel that baseUrls should be exposed there.

Reply:
Okay, I'll switch it around and re-push. I'll also add the same logic to `koa-openapi`

Reply:
sounds good!  if you link locally to get `(?:express|koa)-openapi` working with `openapi-framework`, i can handle the publishing.